cnf-certification-test/networking/suite.go

```diff
@@ -121,8 +121,35 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
 		testPartnerSpecificTCPPorts(&env)
 	})
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestDpdkCPUPinningExecProbe)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
+		testExecProbDenyAtCPUPinning(&env)
+	})
 })
 
+func testExecProbDenyAtCPUPinning(env *provider.TestEnvironment) {
+	ginkgo.By("Check if DPDK pod is running")
+
+	dpdkPods := env.GetDPDKRunningPods()
+	testhelper.SkipIfEmptyAny(ginkgo.Skip, dpdkPods)
+
+	ginkgo.By("Check if cpu pinning is enabled")
+
+	cpuPinningPods := provider.GetCPUPinningPods(dpdkPods)
+	testhelper.SkipIfEmptyAny(ginkgo.Skip, cpuPinningPods)
+
+	ginkgo.By("Check exec probe is not happening")
+
+	for _, cpuPinnedPod := range cpuPinningPods {
```

Comment:
Add a cpuPinningPodsWithDpdk() filter in provider/pods.go to directly get the pods here